src/app/shared/search/search-result-view.model.ts

```diff
@@ -1,7 +1,19 @@
 import { SearchResultViewColumn } from './search-result-view-column.model';
 import { SearchResultViewItem } from './search-result-view-item.model';
+import { CaseResolver } from '../../cases/case.resolver';
+import { Type } from 'class-transformer';
 
 export class SearchResultView {
+  @Type(() => SearchResultViewColumn)
   columns: SearchResultViewColumn[];
+
+  @Type(() => SearchResultViewItem)
   results: SearchResultViewItem[];
+  result_error?: string;
+
+  hasDrafts() {
+    return this.results[0]
+    && this.results[0].case_id
+    && this.results[0].case_id.startsWith(CaseResolver.DRAFT);
```

Comment:
this logic is duplicated. We should find a way to centralise it somewhere

Reply:
what did the business say regarding drafts on the search page?

Reply:
addressed in the next PR https://github.com/hmcts/ccd-case-management-web/pull/183